ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/LastRunStatusState.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ReactElement } from 'react';
+import { ReportStatus } from 'types/report.proto';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import { Flex, FlexItem, Tooltip } from '@patternfly/react-core';
+
+type LastRunStatusStateProps = {
+    reportStatus: ReportStatus;
+};
+
+const errorColor = 'var(--pf-global--danger-color--100)';
+const successColor = 'var(--pf-global--success-color--100)';
+
+const genericMsg =
+    'An issue was encountered. Please try again later. If the issue persists, please contact support';
+
+function LastRunStatusState({ reportStatus }: LastRunStatusStateProps): ReactElement {
+    let statusIcon: ReactElement | null = null;
+    let statusText = '-';
+
+    if (reportStatus.runState === 'SUCCESS') {
+        statusIcon = <CheckCircleIcon color={successColor} title="Success icon" />;
```

Comment:
What do you think about making the titles for these icons more specific to the user case here? Instead of "Success icon", something like "Report run was successful"?

Reply:
Yea, that works for me. I was trying to describe the actual icon being rendered, but maybe it's better to make it describe what it's meant to portray instead